cli/util/cancel.go

```diff
@@ -69,6 +69,18 @@ func cancelTx(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to send conflicting transaction: %w", err), 1)
 	}
+	if ctx.Bool("await") {
+		resTx, _ := c.GetRawTransactionVerbose(resHash)
+		_, err := a.Wait(mainTx.Hash(), mainTx.ValidUntilBlock, nil)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+
+		_, err = a.Wait(resHash, resTx.ValidUntilBlock, nil)
+		if err != nil {
+			return cli.NewExitError(err, 1)
+		}
+	}
 	fmt.Fprintln(ctx.App.Writer, resHash.StringLE())
 	return nil
```

Comment:
You also need a special text to print after conflicting hash. If fallback transaction went in, then print the result of fallback invocation with an appropriate text. If main went in, then print the result of main. If an error is returned and neither of two transactions are in chain (it's possible when main transaction is unknown to the target RPC node, conflicting isn't in chain by the moment of VUB and main is also isn't in chain), then you have to write that conflicting isn't in chain and main as far isn't in chain. The user then may want to retry conflicting tx sending.